py7zr/py7zr.py

```diff
@@ -1228,7 +1228,7 @@ def write(self, fp: BinaryIO, f, assym, folder):
                 insize, foutsize, crc = compressor.compress(fd, fp)
         return self._after_write(insize, foutsize, crc)
 
-    def writestr(self, fp: BinaryIO, f, folder):
+    def writestr(self, f, fp: BinaryIO, folder):
```

Comment:
Should not change an order here because of consistency between `Worker.write` and `Worker.writestr`